tools/mount_gcsfuse/main.go

```diff
@@ -72,89 +100,35 @@ func makeGcsfuseArgs(
 	device string,
 	mountPoint string,
 	opts map[string]string) (args []string, err error) {
+	var flagSet pflag.FlagSet
+	if err := cfg.BuildFlagSet(&flagSet); err != nil {
+		return nil, err
+	}
+
+	boolFlags, nonBoolFlags := flagTypes(&flagSet)
+	nonBoolFlags = append(nonBoolFlags, cfg.ConfigFileFlagName)
+	// TODO: Clean this up after we gain enough confidence on CLI-Config Parity changes.
+	boolFlags = append(boolFlags, "disable-viper-config")
+	noopOptions := []string{"user", "nouser", "auto", "noauto", "_netdev", "no_netdev"}
+
 	// Deal with options.
 	for name, value := range opts {
-		switch strings.ReplaceAll(name, "-", "_") {
-		// Don't pass through options that are relevant to mount(8) but not to
-		// gcsfuse, and that fusermount chokes on with "Invalid argument" on Linux.
-		case "user", "nouser", "auto", "noauto", "_netdev", "no_netdev":
-
-		// Special case: support mount-like formatting for gcsfuse bool flags.
-		case "implicit_dirs",
```

Comment:
Did you find any discrepancy between the flags supported in persistent mounting and the normal flags? Just want to make sure that there are no flags which should not be supported here.

Reply:
I don't think there is any value in doing that. For instance, "foreground" is something that doesn't really seem to make any sense here (correct me if I'm wrong), but it's already there.

Reply:
I've added "o" flag as an exception to ensure that its behavior is the same as before.